src/main/java/pages/HomePage.java

```diff
@@ -20,6 +20,7 @@ public HomePage(WebDriver driver) {
     private static final By CURRENCY_LIST = By.xpath("//*[contains(@class,'hidden-xs') and text()='Currency']");
     private static final By EURO_OPTION = By.xpath("//*[contains(@class,'currency-select') and text()='€ Euro']");
     private static final By TABLET_LIST = By.xpath("//*[contains(@href,'http://opencart.abstracta.us:80/index.php?route=product/category&path=57') and text()='Tablets'][1]");
+    private static final By PHONES_LIST = By.xpath("//*[contains(@href,'http://opencart.abstracta.us:80/index.php?route=product/category&path=24') and text()='Phones & PDAs']");
```

Comment:
This xpath seems to be incorrect, try to enhance it, if you can't and this is the only one you can locate the webelement with, then try to make it like this==> //*[contains(@href,'http://opencart.abstracta.us']